document what needs to be installed to run climatology anomaly example (#501)
cubed-dev/cubed: examples/README.md

```shell
open $(ls -d history/compute-* | tail -1)/timeline.svg
```

## Running the notebook examples

Before running these notebook examples, you will need to install some additional dependencies (besides Cubed).

`mamba install rich pydot flox cubed-xarray`

`cubed-xarray` is necessary to wrap Cubed arrays as Xarray DataArrays or Xarray Datasets.

> `flox` is for supporting more efficient Map Reduce.

Reviewer: ```suggestion
`flox` is for supporting efficient groupby operations in Xarray.
```
